### PR TITLE
Add informations on downstreamed informations

### LINKED ISSUE #227: Docs are not mentioning auth headers in downstream services
Our documentation are not mentioning anywhere that, when an user is authenticated, some headers gets added to the request done to the downstream services automatically for you.

### PR DESCRIPTION
**Pull request by XVincentX**
Raised from some conversations on Gitter.

Connect #227 
Closes #227

### CODE REVIEW

File: docs/consumer-management.md [hunk 1]
```diff
@@ -97,6 +97,15 @@ in order to handle the new credential. You can modify the existing properties, t
 
 We are working on this in order to plug your own credential type.
 
+#### Accessing consumer information from a service enpoint
```
Comment (altsang) [inline docs/consumer-management.md]: `endpoint`

File: docs/consumer-management.md [hunk 2]
```diff
@@ -97,6 +97,15 @@ in order to handle the new credential. You can modify the existing properties, t
...
+
+Express Gateway will add the `eg-consumer-id` header to the proxied requests based on the authenticated user. In case
+there's none, its value will be `anonymous`. Using that ID, your service can query the [Admin API][admin] to retrieve
+more informations.
```
Comment (altsang) [inline docs/consumer-management.md]: `information`

File: docs/consumer-management.md [hunk 3]
```diff
@@ -97,6 +97,15 @@ in order to handle the new credential. You can modify the existing properties, t
...
+
+In case you want to downstream more informations, you can put a [headers][headers] policy just after the authentication
+steps of your pipeline.
```
Comment (altsang) [inline docs/consumer-management.md]: `information`

**Review by altsang — CHANGES_REQUESTED** [review]

**Comment by XVincentX** [comment]
Addressed. Anything else?

**Review by altsang — APPROVED** [review]

**Pull request merged**